bayesian/api_v1.py

```diff
@@ -286,6 +306,11 @@ def get(ecosystem, package, version):
             _session.post(url=METRICS_SERVICE_URL + "/api/v1/prometheus", json=metrics_payload)
 
             return result
+        elif os.environ.get("DISABLE_UNKNOWN_PACKAGE_FLOW", "") == "1":
```

Comment:
instead of using the env variable, you can have a common function in utils or config and then call it. function can be names appropriately.

Reply:
This how other ENV flags were used in this flow, so maintain consistency with current code style.

Reply:
earlier it was in one place. now its in 2 places